eICU_Example/configs/extract_eICU.yaml

```diff
@@ -0,0 +1,36 @@
+defaults:
+  - _extract
+  - _self_
+
+description: |-
+  This pipeline extracts the eICU dataset in longitudinal, sparse form from an input dataset meeting
+  select criteria and converts them to the flattened, MEDS format. You can control the key arguments to this
+  pipeline by setting environment variables:
+  ```bash
+    export EVENT_CONVERSION_CONFIG_FP=# Path to your event conversion config
+    export EICU_PRE_MEDS_DIR=# Path to the output dir of the pre-MEDS step
+    export EICU_MEDS_COHORT_DIR=# Path to where you want the dataset to live
+  ```
+
+# The event conversion configuration file is used throughout the pipeline to define the events to extract.
+event_conversion_config_fp: ${oc.env:EVENT_CONVERSION_CONFIG_FP}
+
+input_dir: ${oc.env:EICU_PRE_MEDS_DIR}
+cohort_dir: ${oc.env:EICU_MEDS_COHORT_DIR}
+
+etl_metadata:
+  dataset_name: eICU
+  dataset_version: 2.0
+
+stage_configs:
+  shard_events:
+    infer_schema_length: 999999999
+
+stages:
+  - shard_events
+  - split_and_shard_subjects
+  - convert_to_sharded_events
+  - merge_to_MEDS_cohort
+  - extract_code_metadata
```

Comment:
I think you could probably delete this stage (just delete the `  - extract_code_metadata` line) and everything would still work as I don't think you have any metadata extraction instructions in your config.

Reply:
This line was removed